src/test_l1_dsVideoDevice.c

```diff
@@ -394,15 +393,18 @@ void test_l1_dsVideoDevice_positive_dsSetDFC (void)
     for (int i = 0; i < gDSvideoDevice_NumVideoDevices; ++i) {
         result = dsGetVideoDevice(i, &handle);
         UT_ASSERT_EQUAL(result, dsERR_NONE);
-        UT_ASSERT_NOT_EQUAL(handle, (intptr_t)NULL);
-        if(gSourceType == 1) {
-            // 03: Set DFC mode with various zoom modes
-            for (int j = 0; j < gDSVideoDeviceConfiguration[j].NoOfSupportedDFCs; ++j) {
-                result = dsSetDFC(handle, gDSVideoDeviceConfiguration[i].SupportedDFCs[j]);
+        // 03: Set DFC mode with various zoom modes
+        for (int j = 0; j < gDSVideoDeviceConfiguration[j].NoOfSupportedDFCs; ++j)
```

Comment:
Update `i`
for (int j = 0; j < gDSVideoDeviceConfiguration[i].NoOfSupportedDFCs; ++j)

Reply:
Done